web-ng/package.json

```diff
@@ -7,13 +7,15 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
+    "eslint": "eslint .",
+    "prettier": "prettier --single-quote --trailing-comma es6 --print-width 80 --write --no-bracket-spacing **/*.js",
```

Comment:
Do we actually need these? Since we're integrating gts, I understood `npm run fix` includes formatting:
https://github.com/google/gts#gts

Reply:
I didn't notice that. Instead, used `eslint` to check for the errors. I'll revert the changes and replace with existing `npm run check` command

Reply:
Is that what you meant?

Reply:
Looks like this was Resolved?

Reply:
Yes